src/test/java/seedu/address/mocks/StorageStub.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.mocks;
+
+import java.nio.file.Path;
+import java.util.Optional;
+
+import seedu.address.model.ReadOnlyEntryBook;
+import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.UserPrefs;
+import seedu.address.storage.Storage;
+
+/**
+ * A mock for Storage for ease of creating objects in tests
+ */
+public class StorageStub implements Storage {
```

Comment:
Good idea! Abstracting a "dummy" storage like this makes tests easier to read with less not-important initialisation code!

Reply:
Thanks :)